<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObjectEditWidget</class>
 <widget class="QGroupBox" name="ObjectEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>276</width>
    <height>231</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>4</number>
     </property>
     <widget class="QWidget" name="pagePlayer">
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="labelPosition">
         <property name="text">
          <string>Position:</string>
         </property>
         <property name="buddy">
          <cstring>coordinatesPlayer</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelPlayerHealth">
         <property name="text">
          <string>Health:</string>
         </property>
         <property name="buddy">
          <cstring>editPlayerHealth</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="editPlayerHealth">
         <property name="toolTip">
          <string>The player's starting health. Normally this is set to full health, 12.</string>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="CoordinateWidget" name="coordinatesPlayer" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageRobot">
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="labelRobotPosition">
         <property name="text">
          <string>Position:</string>
         </property>
         <property name="buddy">
          <cstring>coordinatesRobot</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelRobotHealth">
         <property name="text">
          <string>Health:</string>
         </property>
         <property name="buddy">
          <cstring>editRobotHealth</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="editRobotHealth">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The robot's health. Standard values are:&lt;/p&gt;
&lt;ul&gt;
&lt;li&gt;10 for hoverbots&lt;/li&gt;
&lt;li&gt;20 for rollerbots&lt;/li&gt;
&lt;li&gt;75 for evilbots&lt;/li&gt;
&lt;/ul&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelRobotType">
         <property name="text">
          <string>Type:</string>
         </property>
         <property name="buddy">
          <cstring>comboRobotType</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="comboRobotType"/>
       </item>
       <item row="0" column="1">
        <widget class="CoordinateWidget" name="coordinatesRobot" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageWeapon">
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="labelWeaponPosition">
         <property name="text">
          <string>Position:</string>
         </property>
         <property name="buddy">
          <cstring>coordinatesWeapon</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelWeaponType">
         <property name="text">
          <string>Type:</string>
         </property>
         <property name="buddy">
          <cstring>comboWeaponType</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboWeaponType"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelWeaponAmount">
         <property name="text">
          <string>Amount:</string>
         </property>
         <property name="buddy">
          <cstring>editWeaponAmount</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="editWeaponAmount"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelWeaponContainerSize">
         <property name="text">
          <string>Container Size:</string>
         </property>
         <property name="buddy">
          <cstring>editWeaponContainerWidth</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QSpinBox" name="editWeaponContainerWidth">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Making the container size larger than 1/1 means that searching &amp;lt;i&amp;gt;any&amp;lt;/i&amp;gt; of several tiles let's the player find the item. Use this when placing an item in large objects like desks or beds, so that th player doesn't have to search every single tile of the object.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>128</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="editWeaponContainerHeight">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Making the container size larger than 1/1 means that searching &amp;lt;i&amp;gt;any&amp;lt;/i&amp;gt; of several tiles let's the player find the item. Use this when placing an item in large objects like desks or beds, so that th player doesn't have to search every single tile of the object.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>64</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <widget class="CoordinateWidget" name="coordinatesWeapon" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageDoor">
      <layout class="QFormLayout" name="formLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="labelDoorPosition">
         <property name="text">
          <string>Position:</string>
         </property>
         <property name="buddy">
          <cstring>coordinatesDoor</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelDoorOrientation">
         <property name="text">
          <string>Orientation:</string>
         </property>
         <property name="buddy">
          <cstring>comboDoorOrientation</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboDoorOrientation">
         <item>
          <property name="text">
           <string>Horizontal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Vertical</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelDoorState">
         <property name="text">
          <string>State:</string>
         </property>
         <property name="buddy">
          <cstring>comboDoorState</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="comboDoorState">
         <item>
          <property name="text">
           <string>Opening (A)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Opening (B)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Open</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Closing (A)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Closing (B)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Closed</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelDoorLock">
         <property name="text">
          <string>Lock:</string>
         </property>
         <property name="buddy">
          <cstring>comboDoorLock</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="comboDoorLock">
         <item>
          <property name="text">
           <string>Unlocked</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Locked, ♠ (Spade)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Locked, ♥ (Heart)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Locked, ✷ (Star)</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="CoordinateWidget" name="coordinatesDoor" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageKey">
      <layout class="QFormLayout" name="formLayout_5">
       <item row="0" column="0">
        <widget class="QLabel" name="labelKeyPosition">
         <property name="text">
          <string>Position:</string>
         </property>
         <property name="buddy">
          <cstring>coordinatesKey</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="CoordinateWidget" name="coordinatesKey" native="true"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Unlocks:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,0,0">
         <item>
          <widget class="QRadioButton" name="radioKeySpade">
           <property name="toolTip">
            <string>Spade</string>
           </property>
           <property name="text">
            <string>♠</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioKeyHeart">
           <property name="toolTip">
            <string>Heart</string>
           </property>
           <property name="text">
            <string>♥</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioKeyStar">
           <property name="toolTip">
            <string>Heart</string>
           </property>
           <property name="text">
            <string>✷</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelKeyContainerSize">
         <property name="text">
          <string>Container Size:</string>
         </property>
         <property name="buddy">
          <cstring>editWeaponContainerWidth</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QSpinBox" name="editKeyContainerWidth">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Making the container size larger than 1/1 means that searching &amp;lt;i&amp;gt;any&amp;lt;/i&amp;gt; of several tiles let's the player find the item. Use this when placing an item in large objects like desks or beds, so that th player doesn't have to search every single tile of the object.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>128</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="editKeyContainerHeight">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Making the container size larger than 1/1 means that searching &amp;lt;i&amp;gt;any&amp;lt;/i&amp;gt; of several tiles let's the player find the item. Use this when placing an item in large objects like desks or beds, so that th player doesn't have to search every single tile of the object.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>64</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageTrashCompactor">
      <layout class="QFormLayout" name="formLayout_6">
       <item row="0" column="0">
        <widget class="QLabel" name="labelTrashCompactorPosition">
         <property name="text">
          <string>Position:</string>
         </property>
         <property name="buddy">
          <cstring>coordinatesTrashCompactor</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="CoordinateWidget" name="coordinatesTrashCompactor" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageElevator">
      <layout class="QFormLayout" name="formLayout_7">
       <item row="0" column="0">
        <widget class="QLabel" name="labelElevatorPosition">
         <property name="text">
          <string>Position:</string>
         </property>
         <property name="buddy">
          <cstring>coordinatesElevator</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelElevatorState">
         <property name="text">
          <string>State:</string>
         </property>
         <property name="buddy">
          <cstring>comboElevatorState</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboElevatorState">
         <item>
          <property name="text">
           <string>Opening (A)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Opening (B)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Open</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Closing (A)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Closing (B)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Closed</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelElevatorFloor">
         <property name="text">
          <string>Floor:</string>
         </property>
         <property name="buddy">
          <cstring>editElevatorFloor</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="editElevatorFloor">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="editElevatorTotalFloors">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelElevatorTotalFloors">
         <property name="text">
          <string>Total Floors:</string>
         </property>
         <property name="buddy">
          <cstring>editElevatorTotalFloors</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="CoordinateWidget" name="coordinatesElevator" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageRaft">
      <layout class="QFormLayout" name="formLayout_8">
       <item row="0" column="0">
        <widget class="QLabel" name="labelRaftPosition">
         <property name="text">
          <string>Position:</string>
         </property>
         <property name="buddy">
          <cstring>coordinatesRaft</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="CoordinateWidget" name="coordinatesRaft" native="true"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelRaftDirection">
         <property name="text">
          <string>Direction:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="0,0">
         <item>
          <widget class="QRadioButton" name="radioRaftDirectionLeft">
           <property name="text">
            <string>Left</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioRaftDirectionRight">
           <property name="text">
            <string>Right</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelRaftLeftTurnaround">
         <property name="text">
          <string>Left Stop</string>
         </property>
         <property name="buddy">
          <cstring>editRaftLeftTurnaround</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="editRaftLeftTurnaround">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The left-most x-coordinate of the raft, at which point it turns around.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="maximum">
          <number>127</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelRaftRightTurnaround">
         <property name="text">
          <string>Right Stop:</string>
         </property>
         <property name="buddy">
          <cstring>editRaftRightTurnaround</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="editRaftRightTurnaround">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The right-most x-coordinate of the raft, at which point it turns around.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="maximum">
          <number>127</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageTransporter">
      <layout class="QFormLayout" name="formLayout_9">
       <item row="0" column="0">
        <widget class="QLabel" name="labelTransporterPosition">
         <property name="text">
          <string>Position:</string>
         </property>
         <property name="buddy">
          <cstring>coordinatesTransporter</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelTransporterMode">
         <property name="text">
          <string>Disable:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="checkTransporterDisable">
         <property name="text">
          <string>While robots live</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelTransporterTarget">
         <property name="text">
          <string>Destination:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QRadioButton" name="radioTransporterEOL">
         <property name="text">
          <string>End of map</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QRadioButton" name="radioTransporterCoordinates">
         <property name="text">
          <string>Coordinates</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="LabelTransporterCoordinates">
         <property name="text">
          <string>Coordinates:</string>
         </property>
         <property name="buddy">
          <cstring>coordinatesTransporterDest</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="CoordinateWidget" name="coordinatesTransporter" native="true"/>
       </item>
       <item row="4" column="1">
        <widget class="CoordinateWidget" name="coordinatesTransporterDest" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CoordinateWidget</class>
   <extends>QWidget</extends>
   <header>coordinatewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
